docs/journey/day1/README.md

```diff
@@ -219,6 +219,8 @@ $ kam webhook create \
     --service-name taxi
 ```
 
+Note: If the webhook creation fails with _gitops-webhook-event-listener-route_ route not being present, login to ArgoCD web UI to verify if the apps have been created and synced successfully (instructions on how to access the ArgoCD web UI is at the bottom of this guide)
```

Comment:
```
Note: If the webhook creation fails with _gitops-webhook-event-listener-route_ route not being present, login to ArgoCD web UI to verify if the apps have been created and synced successfully. See link:https://github.com/redhat-developer/kam/blob/master/docs/journey/day1/README.md#visualize-your-applications-on-argocd-ui[instructions] on how to access the ArgoCD web UI.
```
Here providing the link will be intuitive. It will directly take user to the particular instruction location.

Reply:
I think it might make sense to rearrange the instructions on how to access ArgoCD web UI so the instructions are show after
`oc apply -k config/argocd/` is run.

The apps in ArgoCD should be healthy and synced.

Reply:
@wtam2018 @amitkrout I adjusted the ArgoCD Web UI access section [with this commit](https://github.com/redhat-developer/kam/pull/55/commits/e44ea8b3790cbdb8b1708c21d11b9591f1c731dd) based on your suggestion.

Reply:
/approve